client/state/sites/test/selectors.js

```diff
@@ -2466,4 +2467,49 @@ describe( 'selectors', () => {
 			expect( supportsJetpackSettingsUI ).to.be.true;
 		} );
 	} );
+
+	describe( 'getJetpackComputedAttributes()', () => {
+		it( 'should return null for attributes if a site is not Jetpack', () => {
+			const state = {
+				sites: {
+					items: {
+						77203074: {
+							ID: 77203074,
+							URL: 'https://example.com',
+							jetpack: false,
+							options: {
+								jetpack_version: '4.6.0'
+							}
+						}
+					}
+				}
+			};
+
+			const noNewAttributes = getJetpackComputedAttributes( state, 77203074 );
+			expect( noNewAttributes.hasMinimumJetpackVersion ).to.equal( undefined );
+			expect( noNewAttributes.canAutoupdateFiles ).to.equal( undefined );
+			expect( noNewAttributes.canUpdateFiles ).to.equal( undefined );
+		} );
+
+		it( 'should return null for attributes if a site is not Jetpack', () => {
+			const state = {
+				sites: {
+					items: {
+						77203074: {
+							ID: 77203074,
+							URL: 'https://example.com',
+							jetpack: true,
+							options: {
+								jetpack_version: '4.6.0'
+							}
+						}
+					}
+				}
+			};
+			const noNewAttributes = getJetpackComputedAttributes( state, 77203074 );
```

Comment:
I'd use `.to.have.property` here to validate the property existence - I don't think we should be testing the `siteHasMinimumJetpackVersion`, `canJetpackSiteAutoUpdateFiles` and `canJetpackSiteUpdateFiles ` selectors internal behavior - that should be handled respectively by their tests.